src/screens/UserPortal/Chat/Chat.tsx

```
@@ -180,6 +161,25 @@ export default function chat(): JSX.Element {
     });
   }, [selectedContact]);
 
+  React.useEffect(() => {
+    if (filterType === 'all') {
+      chatsListRefetch();
+      if (chatsListData && chatsListData.chatsByUserId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)